### PR TITLE
Update sample to work with latest React Native SDK version

### PR DESCRIPTION
**Pull request by lbalmaceda**
Docs PR: https://github.com/auth0/docs/pull/8348

### CODE REVIEW

**Comment by lbalmaceda** [comment]
@jimmyjames thanks for the review!. 

>add a README in the 00-Login

We've considered in the past doing that, and in fact I think some other libraries do. However all those steps are basically the quickstart itself. It feels like adding that is not gonna end up being "short". I could link to the quickstart on the top of the readme and skip some optional instructions such as "how to change the product bundle identifier of your app". How does that sound?


>Also consider renaming auth0-credentials.js to auth0-credentials.js.example so that developers don't have to dirty their repository to run the application.

The [file is git-ignored](https://github.com/auth0-samples/auth0-react-native-sample/blob/master/00-Login/.gitignore#L1) and shouldn't appear as changed when you change it. Did that happen on your side? It must exists on the repo because that's the way we have to provide variables auto-replacement from the docs site when logged in people click "download sample".

File: 00-Login/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Auth0 React Native Samples - Login
+
+The complete guide to get started with the [react-native-auth0](https://github.com/auth0/react-native-auth0) SDK is on the Auth0 docs link below.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
The complete guide to getting started with [react-native-auth0](https://github.com/auth0/react-native-auth0) is our [React Native QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login).
```

File: 00-Login/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+- [Login QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login)
+
+
+## How to run the sample app
+
+Clone the repository and install the dependencies:
+
+```bash
+yarn install
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
git clone git@github.com:auth0-samples/auth0-react-native-sample.git
yarn install
```

File: 00-Login/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+#### Additional step: iOS
+
+Change the directory into the `ios` folder. You must run the following command to install the SDK pod.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
Change the directory into the `ios` folder and run the following command to install the SDK pod:
```

File: 00-Login/README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+```bash
+# cd ios/
+pod install
+```
+
+You should see the `A0Auth0` pod being installed and linked to the sample app.
+
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
```

File: 00-Login/README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+# Auth0 React Native Samples - Login
+
+The complete guide to get started with the [react-native-auth0](https://github.com/auth0/react-native-auth0) SDK is on the Auth0 docs link below.
+
+- [Login QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login)
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
```

File: 00-Login/README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+# cd ios/
+pod install
+```
+
+You should see the `A0Auth0` pod being installed and linked to the sample app.
+
+
+### Auth0 Credentials 
+
+Open the [Auth0 dashboard](https://manage.auth0.com/dashboard/). Select your existing **Native** type application or create a new one. From the top of the application settings page take the `Client ID` and `Domain` values and put them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library. You can also change and do that manually this way:
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
1. Open the [Auth0 dashboard](https://manage.auth0.com/dashboard/) and click **Applications.**
2. Select your existing Application from the list or click **Create Application** at the top to create a new one of type **Native**. 
3. On the **Settings** tab for the Application, copy the "Client ID" and "Domain" values and paste them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library. You can also change and do that manually this way:
```

File: 00-Login/README.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+```js
+import Auth0 from 'react-native-auth0';
+
+const config = {
+  domain: 'lbalmaceda.auth0.com',
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
  domain: 'YOUR_DOMAIN',
```

File: 00-Login/README.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+  clientId: '4e29mEHknIb1iKesCyeleWJUcz6fVR0O'
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
  clientId: 'YOUR_CLIENT_ID'
```

File: 00-Login/README.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+const auth0 = new Auth0(config);
+```
+
+### Deep linking setup
+The methods required to handle the browser deep link are already in place. If you need to review them, check the full article linked on top of this document.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
The methods required to handle browser deep linking are already in place. If you need to review them, see the [React Native QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login).
```

File: 00-Login/README.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+The URL that the Browser App calls back with the authentication result is created using a combination of the following values.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
The URL that the Browser App calls back with the authentication result is created using a combination of the following values:

```

Comment (joshcanhelp) [reply 00-Login/README.md]: > that the Browser App calls back with the authentication result

Not sure a great replacement but this is awkward.

File: 00-Login/README.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+- package name / product bundle identifier; currently `com.auth0samples` on both platforms.
+- platform name
+- auth0 domain
+
+If you happen to change the first remember to take note of it to update the [Auth0 dashboard](https://manage.auth0.com/dashboard/) configuration later.
```
Comment (joshcanhelp) [inline 00-Login/README.md]: What is a "dashboard configuration?" Do you mean the Application? Which field?

File: 00-Login/README.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+
+#### Additional step: Android
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
#### Additional step for Android apps
```

File: 00-Login/README.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+Open the `android/app/src/main/AndroidManifest.xml` file and locate the Intent Filter declaration. You must update the `android:host` property to use your Auth0 domain from the step above.
+
+```xml
+<data
+  android:host="{DOMAIN}"
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
  android:host="{YOUR_DOMAIN}"
```

File: 00-Login/README.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+  android:pathPrefix="/android/${applicationId}/callback"
+  android:scheme="${applicationId}" />
+```
+
+> The `applicationId` will get auto-completed dynamically.
+
+### Whitelist the URLs
+
+Open one more time the [Auth0 dashboard](https://manage.auth0.com/dashboard/) and select your application. In the "Allowed Callback URLs" add the next 2 values separated with a comma. This is required for the authentication result to be redirected by the browser to the app.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
In the [Auth0 dashboard](https://manage.auth0.com/dashboard/), click **Applications** select your application from the list. In the "Allowed Callback URLs" field, add the 2 values below separated with a comma. This is required for the authentication result to be redirected from the browser to the app.
```

File: 00-Login/README.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+```
+com.auth0samples://lbalmaceda.auth0.com/ios/com.auth0samples/callback,
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback, 
```

File: 00-Login/README.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+> The `applicationId` will get auto-completed dynamically.
+
+### Whitelist the URLs
+
+Open one more time the [Auth0 dashboard](https://manage.auth0.com/dashboard/) and select your application. In the "Allowed Callback URLs" add the next 2 values separated with a comma. This is required for the authentication result to be redirected by the browser to the app.
+
+```
+com.auth0samples://lbalmaceda.auth0.com/ios/com.auth0samples/callback, 
+com.auth0samples://lbalmaceda.auth0.com/android/com.auth0samples/callback,
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
com.auth0samples://YOUR_DOMAIN/android/com.auth0samples/callback,
```

File: 00-Login/README.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+Do the same on the "Allowed Logout URLs". This is required for the browser to redirect to the app after logging you out.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
Add the same values to the "Allowed Logout URLs" field as well. This is required for the browser to redirect to the app after logging you out.
```

File: 00-Login/README.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+
+Remember to click **SAVE** to save the changes.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
Finally, scroll down and click **Save Changes**.
```

File: 00-Login/README.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+
+
+### Run the apps
+
+Run your favorite emulator or simulator, or plug your device.
```
Comment (joshcanhelp) [inline 00-Login/README.md]:
```suggestion
Run your app on an emulator, simulator, or your own connected device.
```

File: 00-Login/README.md (hunk 11 above)
Comment (lbalmaceda) [reply 00-Login/README.md]: I mean the "whitelisting url" section below, which I can ref/link actually. Let me do that.

File: 00-Login/README.md (hunk 16 above)
Comment (lbalmaceda) [reply 00-Login/README.md]: Good. Added the "remember to replace" text

**Pull request merged**